cpp/include/cugraph_c/algorithms.h

```diff
@@ -339,7 +341,7 @@ cugraph_error_code_t cugraph_node2vec(const cugraph_resource_handle_t* handle,
                                       cugraph_graph_t* graph,
                                       const cugraph_type_erased_device_array_view_t* sources,
                                       size_t max_depth,
-                                      bool_t flag_use_padding,
+                                      bool_t compress_result,
```

Comment:
Okay, this is a problem: the semantics of `flag_use_padding` is the _opposite_ of `compress_result`: if you use padding, the result is not compressed; it is compressed when you do _not_ use padding.

Reply:
Yes.  I changed the semantic meaning because it seemed more intuitive to me.  I tend to think of using compression to shrink the result as a behavior I want to enable or disable.

In the implementation I negate the value since, as you observe, it is the opposite of the C++ call.  So I get the correct behavior through this API.

Reply:
Oh, okay, as long as you negate the flag all is well. Thank you!